api/app/utils/cron.ts

```diff
@@ -1,61 +1,110 @@
-import cron from "cron";
+import Queue from "bull";
 
 import { Review } from "../db/models/review";
 import { Transcript } from "../db/models/transcript";
-import { TRANSCRIPT_STATUS } from "../types/transcript";
-import { getUnixTimeFromHours } from "../utils/review.inference";
+import { TranscriptStatus } from "../types/transcript";
+import { buildIsExpiredAndNotArchivedCondition, getUnixTimeFromHours } from "../utils/review.inference";
 import { EXPIRYTIMEINHOURS } from "./constants";
 import { deleteCache } from "../db/helpers/redis";
-import { redis } from "../db";
+import { REDIS_HOST, REDIS_PASSWORD, REDIS_PORT, redis } from "../db";
 
-const CronJob = cron.CronJob;
+const resetRedisCachedPages = async () => {
```

Comment:
Can you put this in the redis helper file? We're reseting the transcript pages in some other places in the transcript controller also

Reply:
sure